README.md

```diff
@@ -9,10 +9,9 @@ https://www.ling.upenn.edu/~kurisuto/germanic/oe_bosworthtoller_about.html
 ## Instructions
 
 ```
-git clone git@github.com:loco-lab/pyglossary.git
+git clone git@github.com:ilius/pyglossary.git
 cd pyglossary
-git checkout at_delim
-pip install .
+python setup.py install
```

Comment:
I prefer `pip install .` as it is tracked properly by `pip` then. Everyone has `pip` anyway. So I think we should leave this line.